test/puppetlabs/trapperkeeper/services/authorization/authorization_service_test.clj

```diff
@@ -213,7 +213,7 @@
           req (assoc catalog-request-nocert :body "Hello World!")
           {:keys [status body]} (app req)]
       (is (= status 403))
-      (is (= body (str "Forbidden request: (127.0.0.1) "
+      (is (= body (str "Forbidden request: 127.0.0.1 "
```

Comment:
The parens only showed up in `actual` because of the inadvertent `nil`<->`""` behavior that I removed.